site/en/docs/extensions/mv3/getstarted/tut-reading-time/index.md

```diff
@@ -0,0 +1,304 @@
+
+## Build the extension {: #build }
+
+To start, create a new directory called `Reading time` that will hold the extension's files. If you
+prefer, you can download the complete source code on [Github][github-reading-time].
+
+### Step 1: Add information about the extension {: #step-1 }
+
+The manifest JSON file is the only required extension file. It holds important information about the
+extension. Create a `manifest.json` file in the _root_ of the project and add the following code:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  "manifest_version": 3,
+  "name": "Reading time",
+  "version": "1.0",
+  "description": "Add the reading time to Chrome Extension documentation articles",
+  ...
+}
+```
+
+These keys contain basic metadata for the extension. It appears in the Extension Management page
+and, when published, in the Chrome Web Store listing. To dive deeper, check out the
+[`"name"`][man-name], [`"version"`][man-ver] and [`"description"`][man-desc] keys on the
+[Manifest][doc-manifest] overview page.
+
+{% Details %}
+{% DetailsSummary %}
+💡 **Other facts about the extension Manifest**
+{% endDetailsSummary %}
+
+- It must be located at the **root** of the project.
+- The only required keys are the `"manifest_version"`, `"name"` and `"version"`.
+- It supports comments (`//`) during development but not in the Chrome Web Store.
+
+{% endDetails %}
+
+### Step 2: Provide the icons {: #step-2 }
+
+So, why do we need icons? Although [icons][doc-icons] are optional during development, they are
+required if you plan to distribute your extension on the Chrome Web Store. They also appear in other
+places like the Extension Management page.
+
+Create an `images` folder and place the icons inside. You can download the icons on
+[Github][github-rt-icons]. Next, add the following code to declare the icons in the manifest:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  ...
+  "icons": {
+    "16": "images/icon-16.png",
+    "32": "images/icon-32.png",
+    "48": "images/icon-48.png",
+    "128": "images/icon-128.png"
+  }
+  ...
+}
+```
+
+We recommend using PNG files, but other file formats are allowed, except for SVG files. Need help
+designing your extension icons? Check out the [icon guidelines and best practices][cws-icons].
+
+{% Details %}
+{% DetailsSummary %}
+💡 **Where are these icon sizes displayed?**
+{% endDetailsSummary %}
+
+| Icon Size | Icon Use                                               |
+|-----------|--------------------------------------------------------|
+| 16x16     | Favicon on the extension's pages and context menu icon.|
+| 32x32     | Windows computers often require this size.             |
+| 48x48     | Displays on the Extension Management page.             |
+| 128x128   | Displays on installation and in the Chrome Web Store.  |
+
+{% endDetails %}
+
+### Step 3: Declare the content script {: #step-3 }
```

Comment:
One of the most common points of confusion I've seen regarding content scripts is that they run in the extension's isolated world. It might help if we briefly mention this concept and link out to another page for a detailed explanation.

Reply:
Good point. I will add a small blurb and link out to [Isolated world](https://developer.chrome.com/docs/extensions/mv3/content_scripts/#isolated_world).